.gitpod.yml

```diff
@@ -0,0 +1,44 @@
+tasks:
+  - init: |
+      npm i &&
+      npm run dx:up &&
+      cp .env.example .env &&
+      set -a; source .env &&
+      sed -i -e "s|http://localhost:3000|$(gp url 3000)|" .env \
+             -e "s|http://localhost:3001|$(gp url 3001)|" .env \
+             -e "s|http://localhost:9000|$(gp url 9000)|" .env 
+    command: npm run d
+
+ports:
+  - port: 3000
+    visibility: public
+    onOpen: open-preview
+  - port: 3001
+    visibility: public
+    onOpen: open-preview
+  - port: 9000
+    visibility: public
+    onOpen: ignore
+  - port: 1100
+    visibility: private
+    onOpen: ignore
+  - port: 2500
+    visibility: private
+    onOpen: ignore
+
+
+github:
+  prebuilds:
+    master: true
+    pullRequests: true
+    pullRequestsFromForks: true
+    addCheck: true
+    addComment: true
+    addBadge: true
+
+vscode:
+  extensions:
+    - bradlc.vscode-tailwindcss
+    - esbenp.prettier-vscode
+    - Prisma.prisma
+    
```

Comment:
- DavidAnson.vscode-markdownlint
    - yzhang.markdown-all-in-one
    - dbaeumer.vscode-eslint
    - ban.spellright

Reply:
Just a tiny change - please add these extensions as well

Reply:
I don't think we need add these extension.

Reply:
I don't think we need add these extension.

Reply:
Hey we have a comprehensive list of extensions at https://github.com/documenso/documenso/blob/feat/refresh/.devcontainer/devcontainer.json#L16

```json
[
  "aaron-bond.better-comments",
  "bradlc.vscode-tailwindcss",
  "dbaeumer.vscode-eslint",
  "esbenp.prettier-vscode",
  "mikestead.dotenv",
  "unifiedjs.vscode-mdx",
  "GitHub.copilot-chat",
  "GitHub.copilot-labs",
  "GitHub.copilot",
  "GitHub.vscode-pull-request-github",
  "Prisma.prisma",
  "VisualStudioExptTeam.vscodeintellicode"
]
```

Reply:
Ok let me add that.